src/laboratory/views/objects.py

```diff
@@ -243,3 +243,20 @@ def view_equipment_list(request, org_pk, lab_pk):
     }
     return render(request, "laboratory/equipment/list.html", context=context)
 
+
+@permission_required('laboratory.view_object')
+def view_material_list(request, org_pk, lab_pk):
+    context = {
+        "org_pk": org_pk,
+        "lab_pk": lab_pk,
+        "laboratory": lab_pk,
+        "create_form": MaterialForm(prefix="create", initial={"type": Object.MATERIAL,
+                                                               "organization": org_pk,
+                                                               "laboratory": lab_pk,
+                                                               "created_by": request.user.pk},
+                                     modal_id="#create_obj_form",
+                                     laboratory_pk=lab_pk),
+        "update_form": MaterialForm(prefix="update", initial={"laboratory": lab_pk},
```

Comment:
Remove the initial parameter in update_form, the instance is already created so is not neccesary to send initial data here.